exercises/variable-length-quantity/.meta/gen.go

```diff
@@ -0,0 +1,115 @@
+// +build ignore
```

Comment:
Ok, can be done with one sweeping commit on all the .meta/gen.go files easily later.